child_compassion/wizard/delegate_child_wizard.py

```diff
@@ -0,0 +1,63 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: David Coninckx <david@coninckx.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import models, fields, api, exceptions, _
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT as DF
+from datetime import datetime
+
+
+class delegate_child_wizard(models.TransientModel):
+    _name = 'delegate.child.wizard'
+
+    partner = fields.Many2one(
+        'res.partner', 'Partner', required=True)
+    comment = fields.Text('Comment', required=True)
+    date_delegation = fields.Date(
+        'Delegation\'s beginning', required=True,
+        default=datetime.today().strftime(DF))
+    date_end_delegation = fields.Date('Delegation\'s end')
+    child_ids = fields.One2many(
+        'compassion.child', string='Selected children',
+        compute='_get_active_ids',
+        default=lambda self: self._get_active_ids())

```

Comment:
Lambda is needed when the function is declared below… I think I will leave it like that because I like to begin the class with the field definitions.